jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -360,6 +361,12 @@ const DeploymentFlow = React.createClass({
       'Button click',
       'Deploy model'
     );
+    if (this.props.stats) {
+      const flags = window.juju_config.flags;
+      const statName = 'deploy' + (flags.ddeploy ? '.direct' : '.target');
```

Comment:
I'm not sure where `ddeploy` is ever set as direct deploy isn't behind a flag, it can be used with the `dd` query param.